### PR TITLE
Fix Amount errors

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: src/main/features/claim/routes/amount.ts [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
 import ErrorHandling from 'common/errorHandling'
+import { ValidationError } from 'class-validator'
 
 function renderView (form: Form<Amount>, res: express.Response): void {
   res.render(Paths.claimAmountPage.associatedView, {
     form: form
   })
 }
 
+let addErrorMessage = function (form: Form<Amount>, fieldName: string, errorMessage: string) {
```
Comment (timja-kainos) [inline src/main/features/claim/routes/amount.ts]: this should be a const

File: src/main/features/claim/routes/amount.ts [hunk 2]
```diff
@@ -22,6 +30,10 @@ export default express.Router()
     ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
       const form: Form<Amount> = req.body
 
+      if (form.model.lowerValue != null && form.model.lowerValue < 0.01) {
+        addErrorMessage(form, 'lowerValue', ValidationErrors.LOWER_VALUE_AMOUNT_NOT_VALID)
```
Comment (timja-kainos) [inline src/main/features/claim/routes/amount.ts]: why not add a min validator to the model?

Comment (dharmendrak) [reply src/main/features/claim/routes/amount.ts]: min makes it mandatory

File: src/main/features/claim/routes/amount.ts (hunk 1 above)
Comment (dharmendrak) [reply src/main/features/claim/routes/amount.ts]: changed to function

File: src/main/features/claim/routes/amount.ts (hunk 2 above)
Comment (timja-kainos) [reply src/main/features/claim/routes/amount.ts]: Can't you just add an `@IsOptional()`

Comment (dharmendrak) [reply src/main/features/claim/routes/amount.ts]: @IsOptional() just check for null

**Pull request merged**